### ISSUE TITLE
Beehive memory leaks???

### ISSUE THREAD

**Issue by JPboane** [FriendsInGlobalHealth/beehive#2]
Merging Mocuba database, after several hours (more that 48 hours) , beehive crashed due the error "Last few GCs". The error was raised doing the UUID duplication hunting on OBS table. This is a huge table which has something like 85.679.586 records in destination database. The source table has 17.533.213 records.
[log_mocuba.log](https://github.com/FriendsInGlobalHealth/beehive/files/5332150/log_mocuba.log)

**Comment by mhawila** [comment]
Yes this is a memory problem [See](https://github.com/nodejs/node/issues/14402) Try running with nodejs version 10 instead. Also we use [JIRA](https://jira.fgh.org.mz/secure/Dashboard.jspa) for problem reporting.

**Comment by JPboane** [comment]
I run it with  --max-old-space-size=4096 and after 33 hours failed again due memory allocation issue
